backend/src/main/scala/cromwell/backend/wdl/ReadLikeFunctions.scala

```diff
@@ -119,14 +119,34 @@ trait ReadLikeFunctions extends PathFactory { this: WdlStandardLibraryFunctions
   override def read_boolean(params: Seq[Try[WdlValue]]): Try[WdlBoolean] =
     read_string(params) map { s => WdlBoolean(java.lang.Boolean.parseBoolean(s.value.trim.toLowerCase)) }
 
+  protected def size(file: WdlValue): Try[Double] = Try(buildPath(file.valueString).size.toDouble)
+
   override def size(params: Seq[Try[WdlValue]]): Try[WdlFloat] = {
```

Comment:
ToL It'd be nice to document the special behavior in a javadoc-style comment here.

I don't know how often Javadoc-style docs are created or perused but I personally like them,